README.md

```diff
@@ -194,6 +194,7 @@ Currently, Exposed is available for **maven/gradle builds**. Check the [Maven Ce
 For more information visit the links below:
 
 -   [Documentation](https://jetbrains.github.io/Exposed/home.html) with examples and docs
+-   [Contributing to Exposed](https://jetbrains.github.io/Exposed/contributing.html)
```

Comment:
Rather than link the docs directly, how about adding an anchor link to the README's [Contributing section](https://github.com/JetBrains/Exposed?tab=readme-ov-file#contributing)?
That section already contains a direct link to the github pages and brings the license into view.